style.css

```diff
@@ -410,3 +410,95 @@ h2 .language {
 .frame-lang img {
   transform: rotate(90deg);
 }
+
+/* ======CONTACT-ME-CSS====== */
+.contact-me {
+  margin-top: 113px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 375px;
+  background-image: url("/images/contactformshapesmobile.svg"), url("/images/Contactme.svg");
+  background-position: 153px 16px, 0 0;
+  background-repeat: no-repeat;
+  height: 779px;
```

Comment:
- [Optional] Please make sure to add this black line at the end of the contact me section.
![image](https://user-images.githubusercontent.com/74744787/214618334-5525d7be-7b01-4ee5-b1d9-f277d11cb8d0.png)
so your code looks exactly like the design 
your design 
![image](https://user-images.githubusercontent.com/74744787/214618549-9354cbcd-2c83-450a-9088-2ec5f0439687.png)

Reply:
thank you and implemented